### ISSUE TITLE
privacy issue with current model: VPN needed

### ISSUE THREAD

**Issue by Shubhambawner** [Swanand01/streaminal#2]
as we know the methods currently used are somehow accessing risky part of the web, why dont we consider adding a vpn like openVPN to the application? as well, why not give an option to keep the downloaded file? ( justice to the data spent in downloading it ;) )

**Comment by Swanand01** [comment]
I don't think we need a VPN, because all streaminal does is download from the magnet link. The malicious part you are referring to (pop ups, redirects) is skipped, since pirate bay's website is never accessed, and it's JS never gets executed.

About the download part, please feel free to take this up. Would be a good addition. WebTorrent CLI has a argument option to allow downloading, maybe we could look into it.

**Comment by Shubhambawner** [comment]
sure, I will try adding download command, and few other things 😊
